pipeline/load.py

```diff
@@ -21,14 +22,100 @@ def create_database_connection(config: _Environ) -> Connection:
     return conn
 
 
+def insert_into_location_table(connection: Connection, plant_data: pd.DataFrame) -> None:
+    """Inserts data from a pandas data frame into a SQL Server into the location table."""
+
+    plant_list = plant_data.values.tolist()
+
+    for plant in plant_list:
+
```

Comment:
Quick question (I'm not sure if you need to) - but does there need to be a "USE plants;" sql string somewhere here?

Reply:
Only the first function being ran requires the use plants. From that point the connection which is the same throughout will be using plants